package.json

```
@@ -71,14 +74,21 @@
         "@types/svgo": "2.6.3",
         "augment-vir": "2.1.0",
         "babel-loader": "8.2.5",
+        "babel-plugin-react-scoped-css": "^1.1.1",
         "http-server": "14.1.0",
         "npm-force-resolutions": "0.0.10",
         "react": "18.1.0",
         "react-dom": "18.1.0",
         "rollup": "2.71.1",
         "rollup-plugin-dts": "4.2.1",
         "rollup-plugin-peer-deps-external": "2.2.4",
+        "rollup-plugin-scss": "^3.0.0",
         "rollup-plugin-terser": "7.0.2",
+        "sass": "^1.50.1",
```

We should stick to a single style for version range prefixes. Imo none is best as it remove all ambiguity:
```suggestion
        "sass": "1.50.1",
```

---

resolved

---

The other new dependencies still have the `^` prefix

---

done